app/views/accounts/_nav_auth.html.erb

```diff
@@ -2,10 +2,15 @@
   <div class="container tablet:padding-x-2 tablet:padding-y-4">
     <div class="clearfix">
       <div class="margin-y-105 margin-left-2 tablet:margin-0 col col-4">
-        <%= link_to image_tag(asset_url('logo.svg'),
-                              alt: APP_NAME,
-                              class: 'text-bottom',
-                              width: 170), root_path %>
+        <%= link_to root_path do %>
```

Comment:
If we're going to do the block form, can we add the optional parens here? I *think* this is where they go?

```suggestion
        <%= link_to(root_path) do %>
```

I also think this is a good candidate to keep with the non-block form?

```
<%= link_to(
  image_tag(
    ...
   ),
  root_url,
) %>
```

Reply:
Yeah, I waffled a bit on how best to indent this. Swapping to block form was a loosely-held feeling that it's more readable as "link to [x]" in being able to determine where X goes without having to read 6 lines further down.

I'll just switch it back and indent enough to appease ERBLint.

Reply:
> I'll just switch it back and indent enough to appease ERBLint.

Updated in eddf355.